src/Sylius/Bundle/ProductBundle/Resources/config/services.xml

```diff
@@ -52,11 +56,14 @@
         <service id="Sylius\Component\Product\Resolver\ProductVariantResolverInterface" class="Sylius\Component\Product\Resolver\CompositeProductVariantResolver">
             <argument type="tagged_iterator" tag="sylius.product_variant_resolver" />
         </service>
+        <service id="sylius.resolver.product_variant.composite" alias="Sylius\Component\Product\Resolver\ProductVariantResolverInterface" />
+        <service id="sylius.resolver.product_variant" alias="Sylius\Component\Product\Resolver\ProductVariantResolverInterface" />
```

Comment:
```suggestion
        <service id="sylius.resolver.product_variant" alias="sylius.resolver.product_variant.composite" />
```
as it will look like that in 2.0